distribution/dashboard/ui/src/components/customModal/ModifyUnitDetailModal.tsx

```diff
@@ -68,12 +68,18 @@ export default function ModifyUnitDetailModal({
         <Form.Item label="LogDiskSize" name="logDiskSize">
           <InputNumber addonAfter={SUFFIX_UNIT} placeholder="请输入" />
         </Form.Item>
-        <Form.Item label="min iops" name="minIops">
-          <InputNumber placeholder="请输入" />
-        </Form.Item>
-        <Form.Item label="max iops" name="maxIops">
-          <InputNumber placeholder="请输入" />
-        </Form.Item>
+        <Row gutter={24}>
+          <Col>
+            <Form.Item label="min iops" name="minIops">
+              <InputNumber placeholder="请输入" />
```

Comment:
There is a problem with the input design of `maxIops` and `minIops` here. It seems that the correlation is relatively poor, and there is no maximum and minimum verification. In fact, it is to input an `Iops` range. It is best to make a range input component, or use `Slider`

Reply:
I will pay attention to this and will adjust it after discussing it with Yu Yi.